crates/biome_js_analyze/src/semantic_analyzers/nursery/no_console.rs

```diff
@@ -0,0 +1,78 @@
+use crate::{semantic_services::Semantic, JsRuleAction};
+use biome_analyze::{
+    context::RuleContext, declare_rule, ActionCategory, FixKind, Rule, RuleDiagnostic, RuleSource,
+};
+use biome_console::markup;
+use biome_diagnostics::Applicability;
+use biome_js_syntax::{global_identifier, AnyJsMemberExpression, JsCallExpression};
+use biome_rowan::{AstNode, BatchMutationExt};
+
+declare_rule! {
+    /// Disallow the use of `console`
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```js,expect_diagnostic
+    /// console.log('hello world')
```

Comment:
Since we already have `noConsoleLog`, maybe it's better to provide a different example for this rule, such as `console.error` and `console.table`